src/utils/src/types/api-utils-types.ts

```diff
@@ -42,21 +42,21 @@ export const enumType = <Items extends Readonly<string[]>>(...items: Items) =>
  * @param bodySchema - The schema describing the body, likely from the bodySchema function
  */
 export const eventSchema = <Body extends Record<string, any>>(bodySchema: Body) =>
-  ({
-    type: 'object',
-    properties: {
-      body: bodySchema,
-      headers: {
-        type: 'object',
-        properties: {
-          'user-agent': stringType
-          // Cookie: stringType
-        },
-        required: ['user-agent']
-      }
-    },
-    required: ['body', 'headers']
-  } as const);
+({
+  type: 'object',
+  properties: {
+    body: bodySchema,
+    headers: {
+      type: 'object',
+      properties: {
+        'user-agent': stringType
+        // Cookie: stringType
+      },
+      required: ['user-agent']
+    }
+  },
+  required: ['body', 'headers']
+} as const);
```

Comment:
I think the formatting here will go back if you save it again with prettier, mind if you check some of the files you made changes to?

Reply:
Yeah, I can go check. I think there's also a command that does all the prettier fixes for you (`prettier ./src --fix` I think? Or something like that) which we should consider adding to the `package.json` scripts so that it's easier to fix these prettier issues in the future. I think there's also one for ESLint, but I'd have to check

Reply:
Yeah, great idea, I submitted a ticket for that